homeassistant/components/upnp/__init__.py

```diff
@@ -195,14 +194,19 @@ async def async_unload_entry(hass: HomeAssistant, config_entry: ConfigEntry) ->
 
 
 @dataclass
-class UpnpSensorEntityDescription(
-    SensorEntityDescription, BinarySensorEntityDescription
-):
+class UpnpBinarySensorEntityDescription(EntityDescription):
     """A class that describes UPnP entities."""
 
     format: str = "s"
 
 
+@dataclass
+class UpnpSensorEntityDescription(EntityDescription):
```

Comment:
```suggestion
class UpnpSensorEntityDescription(SensorEntityDescription):
```
Will be missing an import

Reply:
This was pushed yesterday, not sure why you were seeing older version

Reply:
Was open in a tab from the request yesterday

Reply:
:-) So are we good now?